easybuild/easyconfigs/t/torchtext/torchtext-0.14.1-foss-2022a-PyTorch-1.12.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonPackage'
+
+name = 'torchtext'
+version = '0.14.1'
+local_pytorch_version = '1.12.0'
+local_python_suffix = '-Python-%(pyver)s'
```

Comment:
This is unused, I think it can be removed?

Reply:
It seems deleting the line caused the `eb --update-pr` command to rename the easyconfig file from `torchtext-0.14.1-foss-2022a-PyTorch-1.12.0.eb` to `torchtext-0.14.1-foss-2022a.eb`. So the line was being used after all. Am I wrong?

Reply:
I just made the changes manually (without using the `eb` command) and it seems it will work.